### ISSUE TITLE
Contact page not working

### ISSUE THREAD

**Issue by Almolikey** [jettheme/core#8]
**i tried to copy the script that provided in _the guide of the template_ but i couldn't send a message with it you better try fixing it**

**Comment by mrvancetti** [comment]
> **i tried to copy the script that provided in _the guide of the template_ but i couldn't send a message with it you better try fixing it**

Yes, contact form not working. Were you able to fix this error? I am using at my website but my followers and customers can't send messages. This is bad news :( Please help me...
